### PR TITLE
[airlift] More changes to Airlift tutorial readme

### PR DESCRIPTION
**Pull request by benpankow**
## Summary

Adds a section about fully migrating pipelines off Airflow, and some tweaks based on README runthrough.

### CODE REVIEW

**Comment by benpankow** [comment]
* **#23994** <a href="https://app.graphite.dev/github/pr/dagster-io/dagster/23994?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `master`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @benpankow and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: examples/experimental/dagster-airlift/examples/tutorial-example/migrate.py [hunk 1]
```diff
@@ -40,7 +40,9 @@ def _multi_asset() -> None:
 
 
 def export_duckdb_to_csv_defs(args: ExportDuckDbToCsvArgs) -> Definitions:
-    spec = AssetSpec(key=args.table_name, deps=["customers"])
+    spec = AssetSpec(
```
Comment (benpankow) [inline examples/experimental/dagster-airlift/examples/tutorial-example/migrate.py:43]: ugly, but previous asset key was causing problems - can probably clean up

File: examples/experimental/dagster-airlift/README.md [hunk 2]
```diff
@@ -477,11 +478,10 @@ defs = build_defs_from_airflow_instance(
             "export_customers",
             export_duckdb_to_csv_defs(
                 ExportDuckDbToCsvArgs(
-                    table_name="customers_csv",
+                    table_name="customers",
                     # TODO use env var?
                     csv_path=airflow_dags_path() / "customers.csv",
                     duckdb_path=Path(os.environ["AIRFLOW_HOME"]) / "jaffle_shop.duckdb",
-                    duckdb_schema="raw_data",
```
Comment (benpankow) [inline examples/experimental/dagster-airlift/README.md:484]: outputs of dbt are not stored in schema right now, so had to rm

File: examples/experimental/dagster-airlift/examples/tutorial-example/migrate.py (hunk 1 above)
Comment (schrockn) [reply examples/experimental/dagster-airlift/examples/tutorial-example/migrate.py:43]: Just explicitly pass in the asset spec?

I think that way it is obvious and the user has control

File: examples/experimental/dagster-airlift/examples/tutorial-example/tutorial_example_tests/integration_tests/test_load_dagster.py [hunk 3]
```diff
@@ -1,7 +1,9 @@
+import pytest
 from dagster._core.definitions.definitions_class import Definitions
 
 
 def test_defs_loads(airflow_instance) -> None:
+    pytest.skip("Ben fixes in a follow-up PR")
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/examples/tutorial-example/tutorial_example_tests/integration_tests/test_load_dagster.py:6]: rm

File: examples/experimental/dagster-airlift/examples/tutorial-example/tutorial_example_tests/unit_tests/test_build_dag_defs.py [hunk 4]
```diff
@@ -13,6 +14,7 @@ def dummy_duck_db_path() -> Path:
 
 
 def test_dag_defs_construction_and_node_defs() -> None:
+    pytest.skip("Ben fixes in a follow-up PR")
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/examples/tutorial-example/tutorial_example_tests/unit_tests/test_build_dag_defs.py:17]: rm

File: examples/experimental/dagster-airlift/examples/tutorial-example/migrate.py [hunk 5]
```diff
@@ -0,0 +1,107 @@
...
+from dagster_dbt import DbtProject
+
+# Code also invoked from Airflow
+from tutorial_example.shared.export_duckdb_to_csv import ExportDuckDbToCsvArgs, export_duckdb_to_csv
+from tutorial_example.shared.load_csv_to_duckdb import LoadCsvToDuckDbArgs, load_csv_to_duckdb
+
+
+def dbt_project_path() -> Path:
+    # return Path(__file__).parent / "tutorial_example" / "shared" / "dbt"
+    env_val = os.getenv("TUTORIAL_DBT_PROJECT_DIR")
```
Comment (schrockn) [inline examples/experimental/dagster-airlift/examples/tutorial-example/migrate.py:23]: Are we happy with `TUTORIAL_DBT_PROJECT_DIR`?

**Review by schrockn — DISMISSED** [review]
Make sure to unskip tests.

**Review by schrockn — APPROVED** [review]

**Pull request merged**